esp/services/ws_dfu/ws_dfuService.hpp

```diff
@@ -27,6 +27,7 @@
 #include <atomic>
 #include "ws_dfuHelpers.hpp"
 
+#ifndef _CONTAINERIZED
```

Comment:
I prefer minimizing the number of #ifdefs - because it means there is less likelihood of breaking the code and not spotting it.  I would be inclined to keep this compiled in both - unless there are functions that call code that is not available (e.g. environment).